<pre>1 2016-01-22 14:19:17.826 [info] &lt;0.7.0&gt; Application lager started on node ejabberd2@localhost
<a name='L2'/>2 2016-01-22 14:19:17.877 [info] &lt;0.87.0&gt;@exometer_report:do_start_reporters:614 Starting reporters with []
<a name='L3'/>3 2016-01-22 14:19:17.877 [info] &lt;0.7.0&gt; Application exometer started on node ejabberd2@localhost
<a name='L4'/>4 2016-01-22 14:19:17.932 [info] &lt;0.7.0&gt; Application inets started on node ejabberd2@localhost
<a name='L5'/>5 2016-01-22 14:19:18.089 [info] &lt;0.7.0&gt; Application mnesia started on node ejabberd2@localhost
<a name='L6'/>6 2016-01-22 14:19:18.105 [info] &lt;0.7.0&gt; Application p1_stringprep started on node ejabberd2@localhost
<a name='L7'/>7 2016-01-22 14:19:18.131 [info] &lt;0.7.0&gt; Application sasl started on node ejabberd2@localhost
<a name='L8'/>8 2016-01-22 14:19:18.131 [info] &lt;0.7.0&gt; Application asn1 started on node ejabberd2@localhost
<a name='L9'/>9 2016-01-22 14:19:18.131 [info] &lt;0.7.0&gt; Application public_key started on node ejabberd2@localhost
<a name='L10'/>10 2016-01-22 14:19:18.148 [info] &lt;0.7.0&gt; Application ssl started on node ejabberd2@localhost
<a name='L11'/>11 2016-01-22 14:19:18.157 [notice] &lt;0.65.0&gt;@lager_file_backend:128 Changed loglevel of log/ejabberd.log to info
<a name='L12'/>12 2016-01-22 14:19:18.194 [info] &lt;0.7.0&gt; Application mnesia exited with reason: stopped
<a name='L13'/>13 2016-01-22 14:19:18.287 [info] &lt;0.7.0&gt; Application mnesia started on node ejabberd2@localhost
<a name='L14'/>14 2016-01-22 14:19:18.302 [info] &lt;0.7.0&gt; Application p1_cache_tab started on node ejabberd2@localhost
<a name='L15'/>15 2016-01-22 14:25:57.125 [warning] &lt;0.867.0&gt;@ejabberd_config:reload_local:806 node config reloaded from /home/travis/build/esl/MongooseIM/dev/mongooseim_node2/etc/ejabberd.cfg
<a name='L16'/>16 2016-01-22 14:26:04.338 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 10 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.957.0&gt; exit with reason killed in context shutdown_error
<a name='L17'/>17 2016-01-22 14:26:06.339 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 9 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.955.0&gt; exit with reason killed in context shutdown_error
<a name='L18'/>18 2016-01-22 14:26:08.340 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 8 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.953.0&gt; exit with reason killed in context shutdown_error
<a name='L19'/>19 2016-01-22 14:26:10.341 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 7 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.951.0&gt; exit with reason killed in context shutdown_error
<a name='L20'/>20 2016-01-22 14:26:12.342 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 6 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.949.0&gt; exit with reason killed in context shutdown_error
<a name='L21'/>21 2016-01-22 14:26:14.343 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 5 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.947.0&gt; exit with reason killed in context shutdown_error
<a name='L22'/>22 2016-01-22 14:26:16.344 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 4 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.945.0&gt; exit with reason killed in context shutdown_error
<a name='L23'/>23 2016-01-22 14:26:18.345 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 3 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.943.0&gt; exit with reason killed in context shutdown_error
<a name='L24'/>24 2016-01-22 14:26:20.346 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 2 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.941.0&gt; exit with reason killed in context shutdown_error
<a name='L25'/>25 2016-01-22 14:26:22.347 [error] &lt;0.938.0&gt; Supervisor ejabberd_odbc_sup_localhost had child 1 started with ejabberd_odbc:start_link(&lt;&lt;"localhost"&gt;&gt;, 30000, false) at &lt;0.939.0&gt; exit with reason killed in context shutdown_error
<a name='L26'/>26 2016-01-22 14:26:26.150 [info] &lt;0.7.0&gt; Application lager started on node ejabberd2@localhost
<a name='L27'/>27 2016-01-22 14:26:26.164 [info] &lt;0.90.0&gt;@exometer_report:do_start_reporters:614 Starting reporters with []
<a name='L28'/>28 2016-01-22 14:26:26.164 [info] &lt;0.7.0&gt; Application exometer started on node ejabberd2@localhost
<a name='L29'/>29 2016-01-22 14:26:26.181 [info] &lt;0.7.0&gt; Application inets started on node ejabberd2@localhost
<a name='L30'/>30 2016-01-22 14:26:26.282 [info] &lt;0.7.0&gt; Application mnesia started on node ejabberd2@localhost
<a name='L31'/>31 2016-01-22 14:26:26.317 [info] &lt;0.7.0&gt; Application p1_stringprep started on node ejabberd2@localhost
<a name='L32'/>32 2016-01-22 14:26:26.377 [info] &lt;0.7.0&gt; Application sasl started on node ejabberd2@localhost
<a name='L33'/>33 2016-01-22 14:26:26.377 [info] &lt;0.7.0&gt; Application asn1 started on node ejabberd2@localhost
<a name='L34'/>34 2016-01-22 14:26:26.377 [info] &lt;0.7.0&gt; Application public_key started on node ejabberd2@localhost
<a name='L35'/>35 2016-01-22 14:26:26.398 [info] &lt;0.7.0&gt; Application ssl started on node ejabberd2@localhost
<a name='L36'/>36 2016-01-22 14:26:26.403 [notice] &lt;0.68.0&gt;@lager_file_backend:128 Changed loglevel of log/ejabberd.log to info
<a name='L37'/>37 2016-01-22 14:26:26.428 [info] &lt;0.7.0&gt; Application mnesia exited with reason: stopped
<a name='L38'/>38 2016-01-22 14:26:27.708 [error] &lt;0.650.0&gt; CRASH REPORT Process &lt;0.650.0&gt; with 0 neighbours exited with reason: {aborted,{no_exists,route}} in gen_server:init_it/6 line 330
<a name='L39'/>39 2016-01-22 14:26:27.732 [error] &lt;0.273.0&gt; CRASH REPORT Process &lt;0.273.0&gt; with 0 neighbours exited with reason: bad argument in call to ets:lookup(local_config, {modules,&lt;&lt;"localhost"&gt;&gt;}) in ejabberd_config:get_local_option/1 line 741 in application_master:init/4 line 133
<a name='L40'/><hr/>